fix(Anomalies): Improving the anomalies pages (#12414)
newrelic/docs-website: src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/custom-anomalies.mdx

1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Alerts & AI** and select **Alerts conditions (policies)** in the left navigation panel.

2. Click **+ New alert condition**.

3. Define your signal and click **Next**.

4. Adjust the signal behavior and click **Next**.

5. Click the **Try anomaly thresholds** link.

<img
  width="80%;"
  title="Creating a policy, the **Try anomaly thresholds** link"
  alt="Creating a policy, the **Try anomaly thresholds** link"
  src={alertsTryAnomalyThresholds}
/>

6. After selecting 'anomaly' as your threshold type, you'll need to configure the settings for one or more thresholds. Anomaly detection makes a prediction on what the next data point will be based on prior activity. The threshold value for custom anomaly detection is the number of standard deviations your signal value is away from the value that was predicted.

  To configure the threshold, you'll need to:

    * Set the 'threshold direction' to either upper, lower, or both. This means that we'll only create an incident if the signal value (the output of the query) is above the predicted value, below the predicted value, or either, respectively.

>     This field answers to how many of the data points during a specified time period must be outside of the threshold. The options are **for at least** and **at least once in**. Selecting **for at least** means that ALL of your signal's data points must be outside of the threshold for the specified time period before an incident is opened. The inverse must be true to close the incident. The **at least once in** option simply means that as soon as any of your signal's data points are outside of the threshold, an incident will open. With this option, the time duration is not relevant for determining when to open an incident. However, it's relevant for closing incidents. All of your signal's data points must be within the threshold for the specified period of time

Reviewer: ```suggestion
    This field dictates how many of the data points during a specified time period must be outside of the threshold. The options are **for at least** and **at least once in**. Selecting **for at least** means that ALL of your signal's data points must be outside of the threshold for the specified time period before an incident is opened. The inverse must be true to close the incident. The **at least once in** option simply means that as soon as any of your signal's data points are outside of the threshold, an incident will open. With this option, the time duration is not relevant for determining when to open an incident. However, it's relevant for closing incidents. All of your signal's data points must be within the threshold for the specified period of time
```

    * Set the 'threshold duration'. Think of this as how long the signal value must remain outside the threshold before an incident is opened.   Conversely, it's also how long a signal must be within the threshold for an incident to be closed. 

>     This field answers to the time period mentioned above. It's how long the signal must be in violation of the threshold being defined. This is the actual threshold duration.

Reviewer: ```suggestion
    This field is defined by the time period mentioned above. It's how long the signal must be in violation of the threshold being defined. This is the actual threshold duration.
```

Author: This one I'm not going to add it because it's a passive sentence.
